src/ontology/mondo-edit.obo

```diff
@@ -371884,6 +371884,7 @@ xref: UMLS:C0039075 {source="MONDO:subClassOf", source="ORDO:90025/e", source="N
 is_a: MONDO:0019714 {source="Orphanet:90025"} ! non-syndromic polydactyly, syndactyly and/or hyperphalangy
 is_a: MONDO:0021002 ! syndactyly
 relationship: disease_has_feature HP:0001159 ! Syndactyly
+relationship: excluded_from_qc_check http://purl.obolibrary.org/obo/mondo/sparql/qc/general/qc-single-child.sparql
```

Comment:
non-syndromic syndactyly is a single child of syndactyly, I think that is okay, so I excluded from the QC check.

non-syndromic syndactyly is a grouping class from Orphanet. I am not sure if a syndromic form of syndactyly exists. I imagine there must be syndactyly a feature of syndromes, but it wouldn't be a subclass of syndactyly.

So I guess my question is do you think we should consider obsoleting 'non-syndromic syndactyly'?  I think all the children could just be children of syndactyly.

'non-syndromic syndactyly' is a child of 'non-syndromic polydactyly, syndactyly and/or hyperphalangy' and I wonder if we would want to keep this grouping class.

Children are:
- hyperphalangy (this seems like a phenotype not a disease?)
- 'non-syndromic polydactyly'
- 'non-syndromic syndactyly'

Maybe we don't need any of these grouping classes?

If you agree, we can create a separate ticket for these obsoletions.

Reply:
I agree that hyperphalangy (MONDO:0017455) looks like it is a phenotype. 

@nicolevasilevsky can we discuss this? I am having a hard time understanding what you are asking (maybe it has nothing to do with this PR and that's why I am getting confused). 
I don't know enough about this disease to be able to make a decision regarding whether these grouping classes make sense or not. Having 'non-syndromic syndactyly' makes sense because the parent's definition mention that 'syndactyly can be syndromic or not'

The bigger question is why does "non-syndromic synpolydactyly" is inferred to be a synpolydactyly. 
I think we should review this branch carefully because we have synpolydactyly, syndactyly, syndromic and non-syndromic... 

(Sorry, completely not helpful)

Reply:
I think all of these orphanet groupings will be changed/reclassified when the next Orphanet sync is done, so we probably don't need to worry about this for now.
https://github.com/monarch-initiative/mondo/issues/4579